src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -113,6 +113,7 @@ def send_notification(event, futures):
                         "service_id": service.id,
                     },
                 )
+                return
```

Comment:
@mgaeta I think the better solution is to just re-raise the exception after logging it. We use `safe_excute` at a higher level so things won't break if we throw an exception here (which we know is true because this would throw an exception anyways). I want to get out of the habit of catching errors like this and then returning as if nothing happened.